src/app/onion/learning-object-builder/components/contributor-pill/contributor-pill.component.html

```diff
@@ -1,8 +1,8 @@
 <div class="contributor-pill">
   <div class="contributor-pill__name">
     {{ user.name | titlecase }}
-    <button attr.aria-label="Remove {{ user.name }} as a contributor" class="contributor-pill__dropdown-option" (activate)="removeContributor.emit(user)">
+    <button attr.aria-label="Remove {{ user.name }} as a contributor" class="contributor-pill__dropdown-option" (pointerdown)="removeContributor.emit(user)">
```

Comment:
What happens when you hit tab?

Reply:
it selected the element but pressing space or enter didn't work. I made a change to allow keydown events for enter to allow accessible input. (maybe we should change it to space?)

Reply:
decided to keep enter per offline convo